src/graphorm/index.ts

```diff
@@ -639,6 +639,15 @@ const obj = new GraphORM({
           },
         },
       },
+      pollOptions: {
+        relation: {
+          isMany: true,
+          sort: 'order',
+          order: 'ASC',
+          parentColumn: 'id',
+          childColumn: 'postId',
+        },
+      },
```

Comment:
@sshanzel your additionalQuery would add order to all graphorm queries to PollOption, which we don't always need. From my understanding you wanted to initially sort the pollOptions when requested from post so I would use sort and order from relation on the field like above.

If ok please add a test for this 🙏

Reply:
Thanks for the help! I added two tests. One for feed, and one for single post query.